Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -228,8 +243,14 @@ namespace AZ
             
             PipelineViewMap m_pipelineViewsByTag;
             
-            /// The system time when the last time this pipeline render was started
-            float m_lastRenderStartTime = 0;
+            // The system time when the last time this pipeline render was started
```

Comment:
suggestion: Should this be `The system time when the last pipeline render was started`? (also missing `.`)